FirebaseAuth/Sources/SystemService/FIRAuthStoredUserManager.h

```diff
@@ -62,35 +62,44 @@ NS_ASSUME_NONNULL_BEGIN
 /** @fn getStoredUserForAccessGroup:projectID:error:
     @brief The getter of the user stored locally.
     @param accessGroup The access group to retrieve the user from.
+    @param shareLoginAcrossDevices If true, the keychain will be synced across the end-user's
+   iCloud.
     @param projectIdentifier An identifier of the project that the user associates with. Currently,
         we use API KEY.
     @param outError Return value for any error which occurs.
  */
 - (nullable FIRUser *)getStoredUserForAccessGroup:(NSString *)accessGroup
+                          shareLoginAcrossDevices:(BOOL)shareLoginAcrossDevices
```

Comment:
Same naming feedback re: `shareLoginAcrossDevices` -> `shareAuthStateAcrossDevices` in this file.